config/notifiers.go

```diff
@@ -275,11 +275,14 @@ func (c *WebhookConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 type OpsGenieConfig struct {
 	NotifierConfig `yaml:",inline"`
 
-	APIKey      Secret            `yaml:"api_key"`
-	APIHost     string            `yaml:"api_host"`
-	Description string            `yaml:"description"`
-	Source      string            `yaml:"source"`
-	Details     map[string]string `yaml:"details"`
+	APIKey       Secret            `yaml:"api_key"`
+	APIHost      string            `yaml:"api_host"`
+	Description  string            `yaml:"description"`
+	Source       string            `yaml:"source"`
+	Details      map[string]string `yaml:"details"`
+	Teams        []string          `yaml:"teams"`
+	Tags         []string          `yaml:"tags"`
+	LabelsToTags []string          `yaml:"labels_to_tags"`
```

Comment:
This seems messy to me, as it handles only one use case and doesn't offer any templating.

Could we do something like a delimited string?